package/ubuntu/install-apt-packages.sh

```diff
@@ -4,6 +4,21 @@ set -euo pipefail
 # Update package lists quietly
 sudo apt-get update -qq
 
+# Determine the Ubuntu release version.
+# On Linux Mint, /etc/upstream-release/lsb-release is usually available and contains the original Ubuntu release info.
+if [ -f /etc/upstream-release/lsb-release ]; then
+  ubuntu_release=$(grep '^DISTRIB_RELEASE=' /etc/upstream-release/lsb-release | cut -d'=' -f2)
+else
+  ubuntu_release=$(grep '^VERSION_ID=' /etc/os-release | cut -d'"' -f2)
+fi
+
+# Use VTK7 for Ubuntu releases 22.04 or older; otherwise use VTK9.
+if dpkg --compare-versions "$ubuntu_release" "le" "22.04"; then
```

Comment:
Instead of checking ubuntu version, could it be better to just check if vtk7 or vtk9 is installed?

Reply:
Yes, but defaulting to vtk9 if nothing found.

Reply:
Nvm, I forgot this was an install script and not a CMake setuo